mitmproxy/proxy/protocol/http.py

```diff
@@ -321,6 +321,7 @@ def _process_flow(self, f):
 
         try:
             if websockets.check_handshake(request.headers) and websockets.check_client_version(request.headers):
+                f.metadata['websocket'] = True
```

Comment:
Can't we just use `metadata['websocket_flow']` to identify handshake flows and not add another attribute?